dev-app/android/app/src/main/java/com/dev/app/stripeterminalreactnative/MainApplication.java

```diff
@@ -42,8 +36,13 @@ protected String getJSMainModuleName() {
         }
 
         @Override
-        protected JSIModulePackage getJSIModulePackage() {
-          return new ReanimatedJSIModulePackage();
+        protected boolean isNewArchEnabled() {
+          return BuildConfig.IS_NEW_ARCHITECTURE_ENABLED;
```

Comment:
my understanding is we have it by default set to false https://github.com/stripe/stripe-terminal-react-native/pull/572/files#diff-f33f303916b13a87dedceae01222276bb9c14c05f81452b0c42d2e6ddfe64bd3R33

if we want to support it we would add the line you suggest https://reactnative.dev/docs/the-new-architecture/backward-compatibility-fabric-components#android

Reply:
Ah it's not so much a matter of whether it's set to `true` or `false`, it's more just wondering how the `newArchEnabled` gradle property is being read and propagated as a `BuildConfig` field. It must be being done somewhere else though, otherwise this would be failing to compile 😅

Reply:
https://github.com/facebook/react-native/blob/eb7e10bc0e78d57726e2971c9dc566e9323deb69/packages/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/utils/AgpConfiguratorUtils.kt#L31-L34 looks like its set in react-native-gradle-plugin